cognite_toolkit/cdf_tk/load.py

```diff
@@ -670,8 +670,13 @@ def get_required_capability(cls, ToolGlobals: CDFToolConfig) -> Capability:
     def get_id(self, item: ExtractionPipeline) -> str:
         return item.external_id
 
-    def delete(self, ids: Sequence[str]) -> None:
-        self.client.files.delete(external_id=ids)
+    def delete(self, ids: Sequence[str]) -> int:
+        count = 0
+        for id in ids:
+            with suppress(CogniteAPIError):
+                self.client.extraction_pipelines.delete(external_id=id)
+                count += 1
```

Comment:
this seems a bit expensive? Should there be a similar logic here to CogniteDuplicateError